core/src/main/scala/kafka/coordinator/transaction/TransactionMarkerRequestCompletionHandler.scala

```diff
@@ -34,16 +34,39 @@ class TransactionMarkerRequestCompletionHandler(brokerId: Int,
   override def onComplete(response: ClientResponse): Unit = {
     val correlationId = response.requestHeader.correlationId
     if (response.wasDisconnected) {
-      trace(s"Cancelled request $response due to node ${response.destination} being disconnected")
-      // re-enqueue the markers
+      trace(s"Cancelled write marker request $response due to node ${response.destination} being disconnected")
```

Comment:
The first substitution doesn't look right. I'm not sure we have the request anymore at this point. Maybe we can just print the list of markers?